guacamole/src/main/frontend/src/translations/ca.json

```diff
@@ -406,10 +517,12 @@
         "FIELD_HEADER_TYPESCRIPT_PATH" : "Ruta Typescript :",
         "FIELD_HEADER_USE_SSL"         : "Usa SSL/TLS",
 
+        "FIELD_OPTION_BACKSPACE_EMPTY" : "",
```

Comment:
This need not be included, as it requires no translation.

Reply:
Removed all "_empty" strings